main/index.ts

```diff
@@ -279,7 +279,8 @@ dapps.add({
   openWhenReady: false,
   config: {
     key: 'value'
-  }
+  },
+  status: 'initial'
```

Comment:
does this have any actual consequences?

Reply:
we set it to "initial" in `dapps.add` anyway, this was just to make it conform to the type